codility_training/lessons.lesson09.MaximumSliceProblem.MaxDoubleSliceSum/sAp00n.py

```diff
@@ -0,0 +1,43 @@
+# test case [5, 17, 0, 3], [0, 10, -5, -2, 0], [5, 5, 5] and all negative
+
+def solution(A):
+    if len(A) <= 3:
+        return 0
+    else:
+        B = A[1:-1]
+        print(f'B = {B}')
+        start_ind, end_ind = find_max_slice(B)
+
+        resulte = B[start_ind:end_ind]
```

Comment:
```suggestion
        resulte = find_max_slice(B)
```
함수를 이렇게 만들어보는건 어땠을까요?